src/components/formItem/formItem.svelte

```diff
@@ -71,7 +79,7 @@
     <slot name="label" />
     {#if required}<span class="required-indicator">*</span>{/if}
   </div>
-  <div class="control">
+  <div class="control" bind:this={_controlElement}>
```

Comment:
Out of interest, does it work if we just bind directly to `controlElement`? I think if you change it in the parent, you're asking for trouble, so I'm not too concerned if you get it :laughing:

Reply:
Binding directly to `controlElement` would definitely work. I just preferred not to give the consumer an opportunity to mess it up.

Reply:
Either way is fine - I don't think we can stop people shooting themselves in the foot if they really want, and I don't know what else you'd expect to happen if you change the type of an element (maybe nothing...)

Reply:
Lol, you can't even change the element if you want:
https://svelte.dev/repl/c9c159a994444f62a661a33dbdc24709?version=4.2.8

Reply:
Interesting! Seems to prevent you from completely replacing the element, but you can still manipulate the DOM element itself. Do you see a particular reason why _not_ to do the above, though?

Reply:
I don't think it buys us anything, just makes it slightly harder to follow:
1. You already can't replace the element
2. Even with the indirection, you can still modify the DOM element (as you have a reference to it).

Reply:
Fair point.

Reply:
Either way, feel free to merge!